apps/protailwind/public/sitemap.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
         <sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-        <sitemap><loc>https://protailwind.com/sitemap-0.xml</loc></sitemap>
-<sitemap><loc>https://protailwind.com/server-sitemap.xml</loc></sitemap>
+        <sitemap><loc>https://www.protailwind.com/sitemap-0.xml</loc></sitemap>
```

Comment:
we are having `3XX redirect in sitemap` errors since it goes to a permanent redirect, adding the `www` fixes the error